### ISSUE TITLE
Attributes given by auto annotation is dropped by CVAT

### ISSUE THREAD

**Issue by hyfung** [opencv/cvat#5306]
### My actions before raising this issue
- [x] Read/searched✅ [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched✅ [past issues](/issues)

I am using YoloV5 and PaddleOCR for automatic annotation in CVAT

YoloV5 supports [x,y,w,h,theta] annotation and I wish to add that to my auto annotation

I modified `main.py` to add attributes to the response payload, for example as follows

```
for result in yolo_results_json:
    encoded_results.append({
        'confidence': result['confidence'],
        'label': result['name'],
        'points': [
            result['xmin'],
            result['ymin'],
            result['xmax'],
            result['ymax'],
        ],
        'type': 'rectangle',
        'attributes':[
            {'name': 'number', 'type': 'text', "input_type": "text", 'value': value},
            {'name': 'rotation', 'type': 'number', "input_type": "number", 'value': value},
        ],
    })
```

As to match the response as captured in CVAT
```
[{"confidence":0.9245699048042297,"label":"MY_CLASS","points":[405.3318786621094,0.0,959.914794921875,534.2487182617188],"type":"rectangle","attributes":[]}]
```

For `type` and `input_type` I duplicated them and hoped that CVAT would recognize either of them

Previously I modified `'type': 'rectangle'` to `'type': 'polygon'` and `points` to a list of `[x1,y1, x2, y2, ...]` and it worked, I thought I could do the same to attributes but it did not

### Expected Behaviour

I would expect the attributes passed from auto annotation to be carried forward to the CVAT annotation UI

### Current Behaviour
Nuclio function did return the attributes in the response body

However CVAT dropped the attributes and simply returned a rectangular bounding box, with no rotation nor attributes

### Context
I wish to add attributes support so that auto annotation could also perform OCR and rectangular box rotation at the same time

`~/cvat/cvat/apps/lambda_manager/views.py` does have some code regarding attributes handling, after lambda function invocation

**Comment by yterme** [comment]
Hi, you need to define the attributes in `function.yaml` as explained here [#3903](https://github.com/opencv/cvat/issues/3903) ? That fixed the issue for me. I also had to reload the UI after re-deploying the model for the changes to be visible there.

**Comment by hyfung** [comment]
Thanks @yterme ! That solved this problem

TLDR
- Modify `function.yaml` to add metadata e.g. `{ "id": 0, "name": "trailer_number", "attributes": [{"name": "number", "input_type": "text"}, {"name": "rotation", "input_type": "number"}] }`
- Modify `main.py` of the auto labelling function to also return `attributes` 

```
'attributes':[
  {'name': 'number', 'type': 'text', "input_type": "text", 'value': "45"}
]
```

------
